gazelle/README.md

```diff
@@ -166,27 +163,52 @@ dependencies are added to the `deps` attribute.
 
 ### Tests
 
-Python test files are those ending in `_test.py`.
+A `py_test` target is added to the BUILD file when gazelle encounters
+a file named `__test__.py`.  The following is an example of a `py_test` target
+that gazelle would add.
+
+```starlark
+py_test(
+    name = "build_file_generation_test",
+    srcs = ["__test__.py"],
+    main = "__test__.py",
+    deps = [":build_file_generation"],
+)
+```
+
+Often Python unit test files are named with the prefix `test_` or the suffix `_test`.
+At this point, the plugin does not support generating `py_test` targets for files 
```

Comment:
We merged a change that will generate individual test targets for each file matching this pattern, but only if a `__test__` target is not present.

Reply:
When was it merged?  Cause this is what I could never get to work :(

Reply:
https://github.com/bazelbuild/rules_python/commit/0943375cbbef6b1f73c439b5e350ac93b1b22bfa